packages/clay-modal/stories/Modal.stories.tsx

```diff
@@ -39,7 +39,9 @@ export const Default = (args: any) => {
 					size={args.size}
 					status={args.status}
 				>
-					<ClayModal.Header>{args.title}</ClayModal.Header>
+					<ClayModal.Header clean={args.clean}>
```

Comment:
This demo doesn't make sense. There should be an image in the header?

Reply:
It's probably better in the body, as it seems like adding it to the header adds it to the `<h1>`, but we don't want to add the semantic information of the image to the title

Reply:
I see, putting everything in body is fine too. You will run into an issue where there will be no close button if you don't include an empty `<ClayModal.Header><ClayModal.Header/>`. We would need to implement a close button on `ClayModal.Body`.

Reply:
That's why the idea was to make the `.modal-title` sr-only and then float the close button. The conceptualization exercise was done here: https://codepen.io/marcoscv/pen/azbwEeV
But guys, you know best how to adapt these exercises to a real-world clay implementation!

Reply:
@marcoscv-work I think it's better to have an empty `<ClayModal.Header><ClayModal.Header/>` and don't output `h1` if empty. We can absolute position the close button.